net/grpc/gateway/examples/echo/ts-example/client.ts

```diff
@@ -19,13 +19,8 @@
 import * as grpcWeb from 'grpc-web';
 import * as $ from 'jquery';
 
-// Uncomment either one of the following:
-// Option 1: import_style=commonjs+dts
 import {EchoServiceClient} from './echo_grpc_web_pb';
 
-// Option 2: import_style=typescript
```

Comment:
Thanks for cleaning up!

To test this locally, you could modify the ts-client [Dockerfile](https://github.com/grpc/grpc-web/blob/1efe741695e7d330a4a56fe2e223fc027f83705c/net/grpc/gateway/docker/ts_client/Dockerfile#L23) and manually run the echo server using `docker-compose up --build node-server envoy ts-client`, and see if the example still works (visiting `http://localhost:8081/echotest.html`).

We should probably add integration for the TS import style too soon :)